wiki/api.py

```
@@ -0,0 +1,166 @@
+	for doc in wiki_documents:
+		parent_name = doc["parent_wiki_document"]
+		if parent_name and parent_name in doc_map:
+			doc_map[parent_name]["children"].append(doc_map[doc["name"]])
+		else:
+			root_nodes.append(doc_map[doc["name"]])
+
+	return root_nodes
+
+
+@frappe.whitelist()
+def reorder_wiki_documents(doc_name: str, new_parent: str | None, new_index: int, siblings: str) -> dict:
```

Ruff (0.14.10)

Unused function argument: `new_index`

(ARG001)